upgrade/helpers/rhevm.py

```diff
@@ -262,14 +262,14 @@ def validate_and_create_product_templates():
     if sat_instance and cap_instance:
         sat_host = os.environ.get('RHEV_SAT_HOST')
         new_sat_template = os.environ.get('RHEV_SAT_IMAGE') + "_new"
-        cap_host = os.environ.get('RHEV_CAP_HOST')
+        cap_hosts = os.environ.get('RHEV_CAP_HOST')
```

Comment:
It would be great if we rename it to 'cap_host'. By the way its not a blocker comment.